moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -274,6 +282,107 @@ bool MoveItConfigData::outputOMPLPlanningYAML(const std::string& file_path)
   return true;  // file created successfully
 }
 
+// ******************************************************************************************
+// Output kinematic config files
+// ******************************************************************************************
+bool MoveItConfigData::outputKinematicsYAML(const std::string& file_path)
```

Comment:
@davetcoleman 
I didnt change anything in these functions, does git sees that as a diff because I've added the `setRobotModel` function above?

Reply:
not sure what you mean 'diff'?

Reply:
Looks like you moved them up in the file? I think this is why git sees this as a large diff.

Reply:
Sorry just saw the comment, I mean the difference that `git` saw.